components/rmrk/Gallery/CollectionItem.vue

```diff
@@ -228,6 +229,14 @@ export default class CollectionItem extends mixins(ChainMixin) {
     return !isShareMode
   }
 
+  get showLayoutOptions(): boolean {
+    return this.$store.state.preferences.showLayoutOptions
```

Comment:
use getter

Reply:
Can you explain why this would be necessary here? A getter in Vuex is the counterpart to a computed property, which should be used to evaluate complex logic (e.g. a filtered list or whatever, which isn't the case here).

Reply:
or is it just a convention in this project?

Reply:
> or is it just a convention in this project?

yeah kinda, it's better to keep things consistent as we don't access Vuex state directly from component.
I find that using getters improves readability.

Reply:
> > or is it just a convention in this project?
> 
> yeah kinda, it's better to keep things consistent as we don't access Vuex state directly from component. I find that using getters improves readability.

Gotcha! 😉